apps/prairielearn/src/pages/administratorInstitutions/administratorInstitutions.ts

```diff
@@ -11,7 +12,26 @@ router.get(
   '/',
   asyncHandler(async (req, res) => {
     const institutions = await sqldb.queryRows(sql.select_institutions, InstitutionRowSchema);
-    res.send(AdministratorInstitutions({ institutions, resLocals: res.locals }));
+    const availableTimezones = await getAvailableTimezonesByName();
```

Comment:
Nit: IMO it makes more sense to use `getAvailableTimezones()` here (that is, the function that returns `Timezone[]`).

Reply:
Updated this to getAvailableTimezones() in [ed4f9f4](https://github.com/PrairieLearn/PrairieLearn/pull/9045/commits/ed4f9f44057278fce12a5e2de1948446144ae6c6). Using getAvailableTimezones()` makes this much easier to use!